openfeature/transaction_context/no_op_transaction_context_propagator.py

```diff
@@ -0,0 +1,13 @@
+from openfeature.evaluation_context import EvaluationContext
+from openfeature.transaction_context.transaction_context_propagator import (
+    TransactionContextPropagator,
+)
+
+
+class NoOpTransactionContextPropagator(TransactionContextPropagator):
+    def get_transaction_context(self) -> EvaluationContext:
+        return EvaluationContext()
+
+    def set_transaction_context(
+        self, transaction_context: EvaluationContext
+    ) -> None: ...
```

Comment:
also here to be consistent with the code, just use `pass`
```suggestion
    ) -> None:
    pass
```

`...` in `Protocol` is correct 🙂